services/age_service.js

```diff
@@ -0,0 +1,65 @@
+const ValidationError = require(`../helpers/ValidationError.js`)
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/phiiileo/validate-age/pullRequest?prid=9953536)
```suggestion
const ValidationError = require(`../helpers/ValidationError.js`);
```